openedx/adg/lms/webinars/admin.py

```diff
@@ -58,9 +60,22 @@ class WebinarAdmin(WebinarAdminBase):
     """
     Admin for upcoming and delivered webinars
     """
+    def webinar_status(self, webinar):
+        """
+        Method field to show webinar status.
+
+        Args:
+            webinar (Webinar): Current webinar object
+
+        Returns:
+            string: Webinar status either `Upcoming` or `Delivered`
+        """
+        if not webinar.start_time:
+            return _('Upcoming')
```

Comment:
We will get a 'NoneType' error on "Create Webinar" page. As it is a read-only field so this code will get executed when we visit WebinarAdmin page to create webinar.

Reply:
is there any way to exclude a method field from creating/updation form? I guess if we mark a field as readonly it would be excluded from both of these forms.

Reply:
@muak50 when we visit WebinarAdmin page to create webinar, wouldn't the `webinar` object be `None` at that point? So shouldn't the check be:

```
if not webinar:
    return _('Upcoming')
```

Or am I missing something?

Reply:
@harris-irfan We will have webinar object with default values, so yes webinar object won't be None in this case.

Reply:
@ahmed-faraz46 It's a read-only field. but as per our requirement it will be a part of our form.

Cc: @asadali145

Reply:
Are we showing the `is_cancelled` field to the user? If not then it is okay to display this readonly field. Also test this in case of cloning the webinar.

Reply:
And yes keeping it a part of creation form isn't much of an issue, we can go with both implementations.